movies/src/test/java/comp.fundacionjala/movies/CustomerTest.java

```diff
@@ -0,0 +1,61 @@
+package comp.fundacionjala.movies;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class CustomerTest {
+
+    @Test
+    public void aCustomerInstanceIsNotNull() {
+        Customer customer=new Customer("Juan");
+        assertNotNull(customer);
+    }
+    @Test
+    public void aCustomerInstanceCanBeCreatedGivenACustomerName(){
+        Customer customer=new Customer("Juan");
+        assertTrue(customer instanceof Customer);
+    }
+    @Test
+    public void theNameOfCustomerCanBeExtracted(){
+        Customer customer=new Customer("Juan");
+        assertEquals("Juan",customer.getName());
+    }
+    @Test
+    public void moreThanTwoRentalInstancesCanBeAddedToCustomerShoppingCart(){
+        IMovie movie=new RegularMovie("example");
+        IMovie movie2=new ChildrenMovie("example2");
+        Rental rental=new Rental(movie,7);
+        Rental rental2=new Rental(movie2,4);
+        Customer customer=new Customer("Juan");
```

Comment:
Please Give format to all your classes
Take advantage from your IDE
Press ctrl + shift + alt + L